libbeat/processors/actions/docs/replace.asciidoc

```diff
@@ -5,25 +5,24 @@
 <titleabbrev>replace</titleabbrev>
 ++++
 
-The `replace` processor takes a list of fields to replace the field value
-matching a pattern with replacement string. Under the `fields` key, each entry
-contains a `field: field-name`, `pattern: regex-pattern` and
-`replacement: replacement-string`, where:
+The `replace` processor takes a list of fields to search for a matching
+value and replaces the matching value with a specified string.
 
-* `field` is the original field name. It's supported to use `@metadata.` prefix for the fields and replace values in the event metadata instead of event fields.
-* `pattern` is the regex pattern to match the field's value
-* `replacement` is the replacement string to use to update the field's value
+// Reviewers: The following sentence was in the original doc, but doesn't make sense to me. should I remove it?
+
+//The `replace` processor cannot be used to replace value with a completely new value.
```

Comment:
I agree, it does not make sense. I think what it's supposed to be is:

```suggestion
The `replace` processor cannot be used to create a completely new value.
```

The code reference: 
https://github.com/elastic/beats/blob/26f5d5c0d4ce559b1600dcfdbea0316006ace551/libbeat/processors/actions/replace.go#L99-L119